front/src/common/Map/Layers/GeoJSONs.tsx

```diff
@@ -2,10 +2,11 @@ import { useSelector } from 'react-redux';
 import React, { FC, useEffect, useMemo, useState } from 'react';
 import chroma from 'chroma-js';
 import { Feature, FeatureCollection } from 'geojson';
-import { isPlainObject, keyBy, mapValues, omit } from 'lodash';
+import { isPlainObject, mapValues, omit } from 'lodash';
 import { AnyLayer, Layer, Source, LayerProps } from 'react-map-gl/maplibre';
 import { FilterSpecification } from 'maplibre-gl';
 import { getInfraID } from 'reducers/osrdconf/selectors';
+import { LAYER_ENTITIES_ORDERS } from 'config/layerOrder';
 
 import { Theme } from '../../../types';
 import { RootState } from '../../../reducers';
```

Comment:
```suggestion
import { Theme } from 'types';
import { RootState } from 'reducers';
```

Reply:
I found also other import to make absolute